extensions/pom.xml

```diff
@@ -65,20 +65,6 @@
       <artifactId>guava-testlib</artifactId>
       <scope>test</scope>
     </dependency>
-    <!--
-     | Some extension tests depend on cglib which is not embedded
-     | in an execution that doesn't include package.
-    -->
-    <dependency>
-      <groupId>org.ow2.asm</groupId>
-      <artifactId>asm</artifactId>
-      <scope>test</scope>
-    </dependency>
```

Comment:
should this be kept?

Reply:
Yes, turns out this is needed to support using `mvn test` on the command-line.

Only building up to the `test` phase doesn't do any packaging, so asm is not embedded at this point. Adding asm as a test dependency means you can run extension tests without having built the full core jar. I also fixed a similar issue in `jdk8-tests/pom.xml`: 4cdd15d4620843c4d0f790060d25719d9ac8d6dd